accounts/models.py

```diff
@@ -10,23 +12,24 @@
 """
 class User(AbstractUser):
     email = models.EmailField(unique=True, max_length=50)
-    phone_number = models.CharField(max_length=30, blank=True)
+    phone_number = PhoneNumberField(blank=True, help_text='Contact phone number', null=True)
     verification_code = models.CharField(max_length=100, blank=True)
 
+    
     USERNAME_FIELD = 'email'
-    REQUIRED_FIELDS = ['first_name', 'last_name', 'username']
+    REQUIRED_FIELDS = ['first_name', 'last_name', 'username', 'phone_number'] # add phone number as a requirement while signing up
```

Comment:
This will actually force us to change the field property from ```null=true``` to false. Or what?